src/TorchSharp/NN/Optimizer.cs

```diff
@@ -78,10 +78,40 @@ protected void Dispose (bool disposing)
 
     public partial class Optimizer
     {
+        [DllImport("LibTorchSharp")]
+        private static extern IntPtr THSNN_RMSprop_ctor(IntPtr parameters, int len, double learningRate, double alpha);
+
+        /// <summary>
+        /// Implements RMSprop algorithm.
+        ///
+        /// Proposed by G.Hinton in his course.
+        /// </summary>
+        /// <param name="parameters">Prameters to optimize</param>
+        /// <param name="learningRate">Learning rate (default: 1e-2)</param>
+        /// <param name="alpha">Smoothing constant (default: 0.99)</param>
+        /// <returns></returns>
+        public static Optimizer RMSProp(IEnumerable<TorchTensor> parameters, double learningRate = 0.01, double alpha = 0.99)
```

Comment:
I assume you double checked the default valus with the pytorch docs?

Reply:
Yes.